spec/motion-spec/matchers/satisfy_spec.rb

```diff
@@ -0,0 +1,22 @@
+# -*- encoding : utf-8 -*-
+describe 'Matcher::Satisfy' do
+  it 'satisfy passes when the block returns true' do
+    expect(1).to satisfy { |v| v == 1 }
+  end
+
+  it 'satisfy fails when the block returns false' do
+    expect_failure('Block expected to satisfy condition') do
+      expect(1).to satisfy { |v| v == 3 }
+    end
+  end
+
+  it 'satisfy fails when the block raises an exception' do
+    expect { expect(1).to satisfy { |v| 1 / 0 } }.to raise_error(ZeroDivisionError)
```

Comment:
Unused block argument - `v`. You can omit the argument if you don't care about it.